python/Earthfile

```
@@ -0,0 +1,108 @@
+VERSION 0.6
+FROM ruby:2.7.1
+WORKDIR /dependabot-python
+
+ARG PYTHON_VERSION="3.10.3"
+
+SETUP:
+    COMMAND
+
+    ENV DEPENDABOT_NATIVE_HELPERS_PATH=${DEPENDABOT_NATIVE_HELPERS_PATH:-/opt}
+
+    USER root
+
+    RUN apt-get update && \
+        apt-get install -y --no-install-recommends \
+            libpq-dev \
+            libssl-dev \
+            libbz2-dev \
+            libffi-dev \
+            libreadline-dev \
+            libsqlite3-dev \
+            libcurl4-openssl-dev \
+            llvm \
+            libncurses5-dev \
+            libncursesw5-dev \
+            libmariadb-dev-compat \
+            xz-utils \
+            tk-dev \
+            libxml2-dev \
+            libxmlsec1-dev \
+            libgeos-dev \
+            python3-enchant \
+            python3-pip
```

Missing cleanup step at the end of this command.

```suggestion
            python3-pip \
     && rm -rf /var/lib/apt/lists/*
```